tinyiiod.h

```diff
@@ -26,9 +26,12 @@ struct tinyiiod_ops {
 	/* Read from the input stream */
 	void (*read)(char *buf, size_t len);
 	int32_t (*read_line)(char *buf, size_t len);
+	int32_t (*read_nonbloking)(char *buf, size_t len);
```

Comment:
typo: read_nonbloking -> read_nonblocking

Reply:
I will fix this